vector/src/main/java/im/vector/app/core/extensions/TimelineEvent.kt

```diff
@@ -40,7 +41,8 @@ fun TimelineEvent.getVectorLastMessageContent(): MessageContent? {
     // Iterate on event types which are not part of the matrix sdk, otherwise fallback to the sdk method
     return when (root.getClearType()) {
         VoiceBroadcastConstants.STATE_ROOM_VOICE_BROADCAST_INFO -> {
-            (annotations?.editSummary?.latestContent ?: root.getClearContent()).toModel<MessageVoiceBroadcastInfoContent>()
+            (annotations?.editSummary?.latestEdit?.getClearContent()?.toModel<MessageContent>().toContent().toModel<MessageVoiceBroadcastInfoContent>()
```

Comment:
I wonder if we could replace this by:
`getLastEditNewContent().toModel<MessageVoiceBroadcastInfoContent>()`? The difference is that in this case we would use the `newContent` field instead of `toContent()` extension method.